src/components/AudioPlayer/index.js

```diff
@@ -249,6 +246,14 @@ class AudioPlayer extends Component {
             toggleVolume={this.toggleVolume}
           />
         </View>
+        <SongSlider
+          onSlidingStart={this.onSlidingStart}
```

Comment:
sorry, nit pick, but i think usually the handler function is named `handle...` and not `on...`. So `this.handleSlidingStart` :)